lib/assert.h

```diff
@@ -20,6 +20,9 @@
 void lib_assertPanic(const char *func, int line, const char *fmt, ...);
 
 
+#define LIB_ASSERT_PANIC(fmt, ...) \
```

Comment:
I have two problems with this macro:
- it's not really needed. You can do
```
ret = func();
assert(ret >= 0, "error");
```
instead of
```
if (func() < 0)
 assert("error");
```
- it breaks assert meaning - it's an assert without a condition

Reply:
Done.